cluster-provision/k8s/1.19/provision.sh

```diff
@@ -316,6 +316,24 @@ echo "tmpfs /var/provision/kubevirt.io/tests tmpfs rw,context=system_u:object_r:
 
 dnf install -y NetworkManager-config-server
 
+# Add CNI DHCP daemon service, disabled by default
+cat > /etc/systemd/system/cni-dhcp.service <<EOF
+[Unit]
+Description=CNI DHCP daemon
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+ExecStartPre=rm -f /run/cni/dhcp.sock
+ExecStart=/opt/cni/bin/dhcp daemon
```

Comment:
Interesting, is this deployed on kubevirtci nodes by default? My understanding was that this is installed only with CNAO - so in case user does not install CNAO, there would be a failing service.

Reply:
The service is actually disabled by default, and only enabled on cluster-up when CNAO is enabled

Reply:
@phoracek does the above answer your concern?

Reply:
It does. Sorry I haven't got back to this yet... I will ask somebody from the team to assist with this and https://github.com/kubevirt/kubevirtci/pull/651

Reply:
There is something I am missing here.
The service is suppose to be there already, supplied by `kubernetes-cni.rpm` (which is actuallu suppose to be replaced by `containernetworking-plugins` nowdays).
So why you create/override it here?

I find this strange also because I do not really expect an operator to have to deploy this manually like this.

Reply:
`kubernetes-cni` does not seem to deploy a systemd service (`# repoquery --list kubernetes-cni`).
Grepping around on a CNAO-enabled node, no systemd service seems to have the ability to start the CNI DHCP daemon.
Am I the one missing something? :)
Thanks

Reply:
I think I was too optimistic :)
It appears in [containernetworking-plugins-1.0.0](https://rpmfind.net/linux/RPM/fedora/updates/testing/34/x86_64/Packages/c/containernetworking-plugins-1.0.0-1.fc34.x86_64.html).

I think it did not landed yet on centos-stream, which is a bit odd, because it usually is a bit ahead of fedora.
Can you just make sure it follows those so we can just switch when it lands on centos-stream?

Reply:
Ah, yes, got it!
I will update the PR to use those.
Thanks!

Reply:
Done!

Reply:
Thanks!

Reply:
It seems that containernetworking-plugins v1 is now available for centos steams [link](https://rpmfind.net/linux/RPM/centos/8-stream/appstream/x86_64/Packages/containernetworking-plugins-1.0.0-1.module_el8.5.0+890+6b136101.x86_64.html)
We can get it by updating it as part of provision script, though it doesn't include cni-dhcp service and socket files.
I think its still better to use v1 version.